src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -4841,7 +4843,9 @@ namespace VariableSpeedCoils {
 
                     MixWetBulb = PsyTwbFnTdbWPb(state, MixTemp, MixHumRat, state.dataEnvrn->OutBaroPress, RoutineName);
                     // need to use OutTemp for air-cooled and RatedInletWaterTemp for water-cooled
-                    if (state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).CondenserInletNodeNum != 0) {
+                    if (state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).CondenserInletNodeNum != 0 ||
+                        state.dataVariableSpeedCoils->VarSpeedCoil(DXCoilNum).VSCoilTypeOfNum ==
+                            DataHVACGlobals::Coil_CoolingWaterToAirHPVSEquationFit) {
```

Comment:
Add an exception for variable speed equation fit water-to-air cooling coils so the outdoor air dry-bulb temperature isn't used.